recipe/meta.yaml

```diff
@@ -26,16 +27,25 @@ requirements:
     - six
```

Comment:
`six` has been removed, see https://github.com/plotly/plotly.py/commit/c3fb1a575f009057111f0a7d149861fda2c59129#diff-0b2c14531dd70b010865c2b62e0d87e2896d90de0a2ed420fc0e9ff3b0218787

Reply:
hmm I used the setup.py file from the source code:
install_requires=["plotly", "requests", "retrying>=1.3.3", "six"]

```
 chart-studio-1.1.0 grep -r 'six' ./*
./chart_studio/tools.py:import six
./chart_studio/tools.py:    if isinstance(username, six.string_types):
./chart_studio/tools.py:    if isinstance(api_key, six.string_types):
./chart_studio/tools.py:    if isinstance(proxy_username, six.string_types):
./chart_studio/tools.py:    if isinstance(proxy_password, six.string_types):
./chart_studio/tools.py:    if isinstance(plotly_domain, six.string_types):
./chart_studio/tools.py:    if isinstance(plotly_streaming_domain, six.string_types):
./chart_studio/tools.py:    if isinstance(plotly_api_domain, six.string_types):
./chart_studio/tools.py:    if isinstance(plotly_ssl_verification, (six.string_types, bool)):
./chart_studio/tools.py:    if isinstance(plotly_proxy_authorization, (six.string_types, bool)):
./chart_studio/tools.py:    if isinstance(sharing, six.string_types):
./chart_studio/tools.py:        urlsplit = six.moves.urllib.parse.urlparse(url)
./chart_studio/tools.py:        query_dict = six.moves.urllib.parse.parse_qs(urlsplit.query)
./chart_studio/session.py:import six
./chart_studio/session.py:    "username": six.string_types,
./chart_studio/session.py:    "api_key": six.string_types,
./chart_studio/session.py:    "proxy_username": six.string_types,
./chart_studio/session.py:    "proxy_password": six.string_types,
./chart_studio/session.py:    "plotly_domain": six.string_types,
./chart_studio/session.py:    "plotly_streaming_domain": six.string_types,
./chart_studio/session.py:    "plotly_api_domain": six.string_types,
./chart_studio/session.py:    "sharing": six.string_types,
./chart_studio/session.py:    "filename": six.string_types,
./chart_studio/session.py:    "fileopt": six.string_types,
./chart_studio/session.py:    "sharing": six.string_types,
./chart_studio/plotly/chunked_requests/chunked_request.py:import six
./chart_studio/plotly/chunked_requests/chunked_request.py:from six.moves import http_client
./chart_studio/plotly/chunked_requests/chunked_request.py:from six.moves.urllib.parse import urlparse, unquote
./chart_studio/plotly/chunked_requests/chunked_request.py:        self._bytes = six.b("")
./chart_studio/plotly/chunked_requests/chunked_request.py:                return six.b("")
./chart_studio/plotly/chunked_requests/chunked_request.py:            if _bytes == six.b(""):
./chart_studio/plotly/chunked_requests/chunked_request.py:        if response != six.b(""):
./chart_studio/plotly/chunked_requests/chunked_request.py:                response = six.b("")
./chart_studio/plotly/chunked_requests/chunked_request.py:            self._bytes = six.b("")
./chart_studio/plotly/chunked_requests/chunked_request.py:class _FakeSocket(six.StringIO):
```

Reply:
So pypi chart-studio seems to have the source files that still require six package